proposals/csharp-9.0/native-integers.md

```diff
@@ -92,28 +97,29 @@ The tables below cover the conversions between special types.
 | `nint` | `int` | ExplicitNumeric | `conv.i4` / `conv.ovf.i4` |
 | `nint` | `uint` | ExplicitNumeric | `conv.u4` / `conv.ovf.u4` |
 | `nint` | `long` | ImplicitNumeric | `conv.i8` / `conv.ovf.i8` |
-| `nint` | `ulong` | ExplicitNumeric | `conv.i8` / `conv.ovf.i8` |
+| `nint` | `ulong` | ExplicitNumeric | `conv.i8` / `conv.ovf.u8` |
 | `nint` | `char` | ExplicitNumeric | `conv.u2` / `conv.ovf.u2` |
 | `nint` | `float` | ImplicitNumeric | `conv.r4` |
 | `nint` | `double` | ImplicitNumeric | `conv.r8` |
 | `nint` | `decimal` | ImplicitNumeric | `conv.i8 decimal decimal.op_Implicit(long)` |
 | `nint` | `IntPtr` | Identity | |
 | `nint` | `UIntPtr` | None | |
 | `nint` |Enumeration|ExplicitEnumeration||
+| | | | |
 | `nuint` | `object` | Boxing | `box` |
 | `nuint` | `void*` | PointerToVoid | `conv.u` |
-| `nuint` | `nint` | ExplicitNumeric | `conv.i` / `conv.ovf.i` |
-| `nuint` | `sbyte` | ExplicitNumeric | `conv.i1` / `conv.ovf.i1` |
-| `nuint` | `byte` | ExplicitNumeric | `conv.u1` / `conv.ovf.u1` |
-| `nuint` | `short` | ExplicitNumeric | `conv.i2` / `conv.ovf.i2` |
-| `nuint` | `ushort` | ExplicitNumeric | `conv.u2` / `conv.ovf.u2` |
-| `nuint` | `int` | ExplicitNumeric | `conv.i4` / `conv.ovf.i4` |
-| `nuint` | `uint` | ExplicitNumeric | `conv.u4` / `conv.ovf.u4` |
-| `nuint` | `long` | ExplicitNumeric | `conv.i8` / `conv.ovf.i8` |
-| `nuint` | `ulong` | ImplicitNumeric | `conv.u8` / `conv.ovf.u8` |
+| `nuint` | `nint` | ExplicitNumeric | `conv.i` / `conv.ovf.i.un` |
+| `nuint` | `sbyte` | ExplicitNumeric | `conv.i1` / `conv.ovf.i1.un` |
+| `nuint` | `byte` | ExplicitNumeric | `conv.u1` / `conv.ovf.u1.un` |
+| `nuint` | `short` | ExplicitNumeric | `conv.i2` / `conv.ovf.i2.un` |
+| `nuint` | `ushort` | ExplicitNumeric | `conv.u2` / `conv.ovf.u2.un` |
+| `nuint` | `int` | ExplicitNumeric | `conv.i4` / `conv.ovf.i4.un` |
+| `nuint` | `uint` | ExplicitNumeric | `conv.u4` / `conv.ovf.u4.un` |
+| `nuint` | `long` | ExplicitNumeric | `conv.u8` / `conv.ovf.i8.un` |
+| `nuint` | `ulong` | ImplicitNumeric | `conv.u8` / `conv.u8` |
 | `nuint` | `char` | ExplicitNumeric | `conv.u2` / `conv.ovf.u2.un` |
-| `nuint` | `float` | ImplicitNumeric | `conv.r.un conv.r4` |
-| `nuint` | `double` | ImplicitNumeric | `conv.r.un conv.r8` |
+| `nuint` | `float` | ImplicitNumeric | `conv.r4` |
+| `nuint` | `double` | ImplicitNumeric | `conv.r8` |
```

Comment:
Let's keep the `conv.r.un` for these cases, since that is needed (see https://github.com/dotnet/roslyn/issues/60714). (I didn't realize the table included `conv.r.un` when we discussed this earlier.)

Reply:
Restored